be/src/storage/rowset/segment_v2/bitshuffle_page.h

```diff
@@ -374,8 +374,16 @@ class BitShufflePageDecoder final : public PageDecoder {
     EncodingTypePB encoding_type() const override { return BIT_SHUFFLE; }
 
 private:
+    const void* get_data(size_t pos) {
```

Comment:
use inline function to improve the performance.

Reply:
> 

done, https://github.com/StarRocks/starrocks/pull/1597/commits/2484fe9e0fd6d0e9cf1c9dd1c677ec4d2e87d2e1

Reply:
> 

A member-function defined inside a class declaration is implicitly an inline function, so actually, there is no need to add an `inline` keyword.  @chaoyli